xAPI.md

```diff
@@ -2654,7 +2646,7 @@ The following table lists xAPI scope values:
 
 ###### OAuth Extended Parameters  
 Note that the parameters "consumer_name" and "scope" are not part of 
-OAuth 1.0, and therefore if used should be passed as query string or form 
+OAuth 1.0, and therefore if used MUST be passed as query string or form 
```

Comment:
Hmm, let me see... interesting. They *could* be included in the OAuth headers per the OAuth 1.0 spec (http://tools.ietf.org/html/rfc5849 , "Servers MAY specify additional parameters."). But, since we're the ones adding them with the spec and get to define their behavior and this was already in here, I think MUST is fine.